src/components/format-date/format-date.test.ts

```diff
@@ -88,14 +90,15 @@ describe('<sl-format-date>', () => {
       it(`date has correct era format: ${eraFormat}`, async () => {
         const el = await fixture<SlFormatDate>(
           html`
-            <sl-format-date .date="${new Date(new Date().getFullYear(), 0, 1)}" era="${eraFormat}"></sl-format-date>
+            <sl-format-date date="${new Date(new Date().getFullYear(), 0, 1)}" era="${eraFormat}"></sl-format-date>
           `
         );
 
         const expected = new Intl.DateTimeFormat('en-US', { era: eraFormat }).format(
           new Date(new Date().getFullYear(), 0, 1)
         );
-        expect(el.shadowRoot?.textContent?.trim()).to.equal(expected);
+        const innerTimeEl = el.shadowRoot?.querySelector('time');
```

Comment:
What's the difference between this and the original? They'll both produce the same result, as `textContent` will strip the `<time>` tag. Seems like extra code.